file.json

```diff
@@ -0,0 +1 @@
+{"City.58c426c0-75b5-49e6-a2b0-de3285b60709": {"__class__": "City", "updated_at": "2018-08-20T23:16:37.025331", "id": "58c426c0-75b5-49e6-a2b0-de3285b60709", "created_at": "2018-08-20T23:16:37.025285"}, "State.fd0234f0-bedc-424b-968f-d3eb109ece2a": {"__class__": "State", "updated_at": "2018-08-20T23:16:32.136463", "id": "fd0234f0-bedc-424b-968f-d3eb109ece2a", "created_at": "2018-08-20T23:16:32.136397"}}
```

Comment:
This file is unnecessary to have in our repo, as it changes over time, but for now, let's keep it.